app/lib/pdf_filler/id39r_pdf.rb

```diff
@@ -16,18 +16,19 @@ def initialize(submission)
 
     def hash_for_pdf
       answers = {
+        "BL6" => @xml_document.at('Form39R ChildCareCreditAmt')&.text,
```

Comment:
@powersurge360 how come you included the `Form39R` here, as opposed to the other selectors `IncomeUSObligations` and `HealthInsurancePaid` fields?

Reply:
I don't think you strictly need it if there is only one particular `ChildCareCreditAmt`. I mostly did it for clarity and specificity I suppose.